README.md

```diff
@@ -1,2 +1,151 @@
+RFC are:
+
+   - A new feature that creates new API surface area
+   - The removal of features that already shipped
+   - The introduction of new idiomatic usage or conventions
+
+The RFC process is a great opportunity to get more eyeballs on your proposal
+before it becomes a part of a released version of Salt. Quite often, even
+proposals that seem "obvious" can be significantly improved once a wider group
+of interested people have a chance to weigh in.
+
+The RFC process can also be helpful to encourage discussions about a proposed
+feature as it is being designed, and incorporate important constraints into the
+design while it's easier to change, before the design has been fully
+implemented.
+
+Changes that do **NOT** require an RFC:
+
+  - Rephrasing, reorganizing or refactoring
+  - Bug fixes
+  - Addition or removal of warnings
+  - Additions only likely to be _noticed by_ other implementors-of-Salt,
+    invisible to users-of-Salt.
+
+## What the process is
+
+In short, to get a major feature added to Salt, one must submit the RFC via
+pull-request. After a comment period, the RFC will be either **Accepted** or
+**Rejected**. An **Accepted** RFC may then be implemented with the goal of
+eventual inclusion into Salt.
+
+## The RFC life-cycle
+The following is a more detailed explanation of the process.
+
+### 1. Proposal
+The RFC is proposed by submitting a pull request to this repo, by copying
+`0000-template.md` and modifying it. If the RFC pertains to any open issues,
+reference them in the **Salt Issue(s)** entry.
+
+When copying the file, do _not_ assign it a number. Simply name the file with a
+short description of the RFC (e.g. `subspace-transport.md`). Once the pull
+request has been opened, a SaltStack core engineer will assign the RFC a number
+and the RFC will enter the initial **Draft** status. At this time, the
+following changes can be made to the RFC file:
+
+1. Add the pull request number to the **RFC PR** entry at the top of the file
+2. Rename the file to include the assigned RFC number, then commit and push to
+   update the pull request
+    ```bash
+    $ git mv subspace-transport.md 0123-subspace-transport.md
+    $ git commit -am 'Assigned RFC number'
+    $ git push origin branchname
+    ```
+
+### 2. Discussion
+The pull request will remain open and serve as the comment thread for the RFC.
+The initial comment period will last no fewer than two (2) weeks, and may be
+extended as deemed necessary based on comment activity.
+
+Once the initial comment period has ended, the RFC will enter **Final Comment**
+status. One (1) week will be allowed for any further comments. At the end of
+the **Final Comment** period, a decision will be made on whether or not to
+accept the RFC. Acceptance requires approval from five (5) members of the core
+development team. As project creator, [Thomas
+Hatch](https://github.com/thatch45) will have a final veto on any RFC.
```

Comment:
>I’m asking the interested parties to look at the mechanics of this proposal and provide feedback on if it will help enable them to have more insight and better mechanisms to work with the core team to improve Salt and ensure concerns and complaints are surfaced and addressed before development begins, and if not where can that be improved so that we can establish this starting point. Then allow us to use this process to elicit feedback on other important matters of the project as well as the process.

For the mechanics of the proposal, I would like to propose that we discuss an alternative to how RFCs get accepted.

>In short, I think that much of what needs to be achieved here could easily be done so through the formation of an RFC Review Team that would consist of a balance of community members and Salt employees or customers. The community side could be elected through a process that ought not to be too hard to enact.  
>
>The team could meet every couple of weeks in an open meeting held via videoconference which is announced well in advance. Anybody not formally on the RFC Review Team who wishes to would be welcome to join would, of course, be welcome.
>
>Agendas would typically be to review open RFCs and to provide comments and discussion. For RFCs ready for a vote, members could vote. The BDFL could, as discussed, veto any decision though this would likely be quite rare.
>
>Meeting minutes and and video archives could be made available to those who could not attend.

Reply:
I think I'd like to see the current proposal to through a couple of PRs before requiring a synchronous process.

Salt, being a world wide community, has interested parties across time zones.

As it is, discussion already can happen in the open on PRs - if we feel the need to summarize discussion points, that can happen on the PR. The only thing we might lose is the face-to-face nature of the teleconference.

I do prefer face-to-face interactions for some things (there's a reason there's a language summit during pycons), but I'm not sure if it's the best choice for typical RFCs.

Do you have some experience using video conferencing for reviews, or some other thoughts about why it would be better than the asynchronous style proposed?

Reply:
Hi @waynew. I took a brief swing through some popular projects to find a couple of quick examples for you:

https://github.com/helm/helm 
Developer Call: Thursdays at 9:30-10:00 Pacific. https://zoom.us/j/696660622

Jaeger tracing
https://www.jaegertracing.io/get-in-touch/#bi-weekly-project-meetings

Envoy:
https://github.com/envoyproxy/envoy#community-meeting

I honestly just grabbed three projects totally at random from the CNCF page but there are many others.